docs/vanilla.md

```diff
@@ -35,32 +35,138 @@ of VDDK Programming Guide. **Note: please apply privileges at the vCenter Server
 
 ```bash
 velero plugin add <plugin-image>
 ```
 
-For Version 1.1.1 the command is
+For Version 1.3.0 the command is
 
 ```bash
-velero plugin add vsphereveleroplugin/velero-plugin-for-vsphere:v1.1.1
+velero plugin add vsphereveleroplugin/velero-plugin-for-vsphere:v1.3.0
 ```
 
-Please refer to [velero-plugin-for-vsphere tags](https://github.com/vmware-tanzu/velero-plugin-for-vsphere/tags) for correct tags for versions 1.1.1 and higher.
+Please refer to [velero-plugin-for-vsphere tags](https://github.com/vmware-tanzu/velero-plugin-for-vsphere/tags) for correct tags for versions 1.3.0 and higher.
 
 #### Install in Air-gapped environment
 
 If it is an air-gapped environment, please refer to [Install Notes with Customized Images](install-notes.md) for extra required steps.
 
+## Upgrade
+
+An alternate to upgrading would be to [Uninstall](#uninstall) and do a fresh [Install](#install)
+
+1. [Scale down Velero and plugin components](#scale-down-velero-and-plugin-components)
+2. [Create vSphere Credentials Secret](#create-vsphere-credentials-secret)
+3. [Create Velero vSphere Plugin Configuration](#create-velero-vsphere-plugin-configuration)
+4. [Update the plugin image](#update-the-plugin-image)
+5. [Scale up velero deployment](#scale-up-velero-deployment)
+
+#### Scale down Velero and plugin components
+
+The following command scales down the `velero` deployment and deletes `backup-driver` and `datamgr-for-vsphere-plugin`
+
+```bash
+kubectl -n <velero-namespace> delete deploy/backup-driver
+kubectl -n <velero-namespace> delete daemonset datamgr-for-vsphere-plugin
+kubectl -n <velero-namespace> scale deploy/velero --replicas=0
+```
+
+### Create vSphere Credentials Secret
+
+You can retrieve these values from Secret `vsphere-config-secret` in `vmware-system-csi` namespace for
+2.3 vSphere CSI driver OR Secret `vsphere-config-secret` in `kube-system` namespace for 2.2 vSphere CSI
+driver
+
+```bash
+% cat csi-vsphere.conf
+[Global]
+cluster-id = "cluster1"
+
+[VirtualCenter "10.182.1.133"]
+user = "Administrator@vsphere.local"
+password = "password"
+port = "443"
+
+% kubectl -n <velero-namespace> create secret generic velero-vsphere-config-secret --from-file=csi-vsphere.conf
+```
+
+### Create Velero vSphere Plugin Configuration
+
+The config map indicates to the plugin the cluster flavor and VC Credential Secret created in the previous step
+
+```bash
+% cat <<EOF | kubectl -n <velero-namespace> apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: velero-vsphere-plugin-config
+data:
+  cluster_flavor: VANILLA
+  vsphere_secret_name: velero-vsphere-config-secret
+  vsphere_secret_namespace: <velero-namespace>
+EOF
+```
+
+### Update the plugin image
+
+```bash
+kubectl -n velero set image deployment/velero velero-plugin-for-vsphere=vsphereveleroplugin/velero-plugin-for-vsphere:v1.3.0
+```
+
+### Scale up Velero deployment
+
+```bash
+kubectl -n <velero-namespace> scale deploy/velero --replicas=1
+```
+
```

Comment:
Do we need to say something about Supervisor in the supervisor doc section?

Reply:
no changes are necessary for supervisor install.